src/models/fhir/constraints.ts

```diff
@@ -128,6 +128,9 @@ const artGroupedFhirKeys = {
   "observations._.device.type.system": "_.Device.type.coding[0].system",
   "observations._.device.type.code": "_.Device.type.coding[0].code",
   "observations._.device.type.display": "_.Device.type.coding[0].display",
+
+  // Note(s)
+  "observations._.observation.note": "_.Observation.note",
```

Comment:
With reference from the [mapping table](https://github.com/Notarise-gov-sg/api-notarise-healthcerts/wiki/2.-Compose-and-validate-fhirBundle#mapping-table), let's do this:

```suggestion
  "observations._.observation.modality": "_.Observation.note[0].{ id=MODALITY, text }",
```